docs/gettingstarted/index.md

```diff
@@ -1,60 +1,119 @@
+
+```gradle
+// In some build.gradle
+group = 'com.example'
+```
+
+The packages within your java source (`src/main/java`) should also now conform to this structure, with an inner package representing the mod id:
+
+```text
+com
+- example (group package)
+  - mymod (the mod id)
+    - MyMod.java (renamed ExampleMod.java)
+```
+
+#### Version
+
+Set the `version` property to the current version of your mod. We recommend using a [variation of semantic versioning][semver].
+
+```gradle
+// In some build.gradle
+version = '1.19.4-1.0.0.0'
+```
+
+### Additional Configurations
+
+Additional configurations can be found on the [ForgeGradle] docs.
 
 Building and Testing Your Mod
 -----------------------------
 
-1. To build your mod, run `gradlew build`. This will output a file in `build/libs` with the name `[archivesBaseName]-[version].jar`. This file can be placed in the `mods` folder of a Forge enabled Minecraft setup or distributed.
-2. To test run your mod, the easiest way is to use the run configs that were generated when you set up your project. Otherwise, you can run `gradlew runClient`. This will launch Minecraft from the `<runDir>` location along with your mod's code in any source sets specified within your run configurations. The default MDK includes the `main` source set, so any code written within `src/main/java` will be applied.
-3. You can also run a dedicated server using the server run config or via `gradlew runServer`. This will launch the Minecraft server with its GUI. After the first run, the server will shut down immediately until the Minecraft EULA is accepted by editing `run/eula.txt`. Once accepted, the server will load and can be accessed via a direct connect to `localhost`.
+1. To build your mod, run `gradlew build`. This will output a file in `build/libs` with the name `[archivesBaseName]-[version].jar`. This file can be placed in the `mods` folder of a Forge-enabled Minecraft setup or distributed.
+1. To run your mod in a test environment, you can either use the generated run configurations or use the associated tasks (e.g. `gradlew runClient`). This will launch Minecraft from the run directory (default 'run') along with any source sets specified. The default MDK includes the `main` source set, so any code written in `src/main/java` will be applied.
+1. If you are running a dedicated server, whether through the run configuration or `gradlew runServer`, the server will initially shut down immediately. You will need to accept the Minecraft EULA by editing the `eula.txt` file in the run directory. Once accepted, the server will load, which can then be accessed via a direct connect to `localhost`.
 
 !!! note
-    It is always advisable to test your mod in a dedicated server environment if it is intended to run there.
-    
+    You should always test your mod in a dedicated server environment. This includes [client-only mods][client] as they should not do anything when loaded on the server.
```

Comment:
This could maybe explain best practices for client-only mods (such as client-only EBS or early constructor exit with a different method). Could be in a different page or [here](https://github.com/MinecraftForge/Documentation/pull/490/files#diff-2099cf892b927b6c29c6c1190c139f16521d5b87764b74ad0bb6b1457060dc8dR54).

Reply:
I'm not sure how to best handle the organizational structure here since we have a 'sides' page which should be dedicated to these question and is linked to from this page. I think the best idea is to put that on the backburner until I rewrite sides with more examples, which I should be able to tackle in a separate PR relatively quickly.